### PR TITLE
WIP - Adding three columns content option

### PR DESCRIPTION
**Pull request by burakukula**
screen shot 2018-04-03 at 11 57 33

We need it here:

screen shot 2018-04-03 at 11 40 30

### CODE REVIEW

File: src/components/layout/iframe-pages/layout_three_columns.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      <style>{'body {background-color: #f0f3f5;}'}</style>
+    </head>
+    <body>
+      <Layout threeColumns header={header} footer={footer}>
+        <LayoutLeftContent>
+          <LayoutBox>
+            <Text>
+              Left content
+            </Text>
+          </LayoutBox>
```
Comment (tomasz-sodzawiczny) [inline src/components/layout/iframe-pages/layout_three_columns.jsx]: Remove the padding and background, it's not the way we'll use it, it's also confusing when you look at the example (the right column explicitaly says *"Aside content has no padding and no background"*).

Comment (burakukula) [reply src/components/layout/iframe-pages/layout_three_columns.jsx]: @tomasz-sodzawiczny ups. On my way. thx

File: src/components/layout/Layout.jsx [hunk 2]
```diff
@@ -32,6 +44,7 @@ Layout.propTypes = {
   header: PropTypes.node,
   footer: PropTypes.node,
   reversedOrder: PropTypes.bool,
+  threeColumns: PropTypes.bool,
```
Comment (tomasz-sodzawiczny) [inline src/components/layout/Layout.jsx]: How do we pass the third column in?

File: src/components/layout/Layout.spec.jsx [hunk 3]
```diff
@@ -130,3 +141,15 @@ describe('LayoutBox', () => {
     expect(layoutBox.hasClass('sg-layout__box')).toEqual(true);
   });
 });
+
+describe('LayoutLeftContent', () => {
```
Comment (tomasz-sodzawiczny) [inline src/components/layout/Layout.spec.jsx]: Do we always render the contents of the left-most column? What happens on small screens?
```
L C R
```
v
```
L
C
R
```
?

Comment (burakukula) [reply src/components/layout/Layout.spec.jsx]: @tomasz-sodzawiczny we should display:

```
L
C
R
```

**Pull request closed without merging**